client/app/views/register/my_accounts.coffee

```diff
@@ -3,6 +3,11 @@
 module.exports = class MyAccountsView extends LayoutView
```

Comment:
Is it wanted to put this file inside the `register` folder here? I would put it int the `views/subviews/` instead. What do you think?